Composer/cypress/integration/Breadcrumb.spec.ts

```diff
@@ -35,15 +35,15 @@ context('breadcrumb', () => {
     hasBreadcrumbItems(cy, ['__TestTodoSample']);
   });
 
-  it('can show event name in breadcrumb', () => {
+  it('can show dialog and trigger name in breadcrumb', () => {
     cy.findByTestId('ProjectTree').within(() => {
       cy.findByTestId('addtodo_Dialog started').click();
     });
 
-    hasBreadcrumbItems(cy, ['__TestTodoSample', 'Dialog started']);
+    hasBreadcrumbItems(cy, ['addtodo', 'Dialog started']);
   });
 
-  it('can show action name in breadcrumb', () => {
+  it.skip('can show action name in breadcrumb', () => {
```

Comment:
Can remove the skip

Reply:
This has been removed; it's running all 3 tests now.